src/Hedgehog.Experimental/Gen.fs

```diff
@@ -24,8 +24,16 @@ type AutoGenConfig = {
   Uri : Gen<Uri>
   SeqRange : Range<int>
   RecursionDepth: int
+  Overrides: Map<string, Gen<obj>>
 }
 
+
+module AutoGenConfig =
+
+  let addOverride (gen: Gen<'a>) config =
+    { config with Overrides = config.Overrides.Add (typeof<'a>.FullName, Gen.map box gen) }
```

Comment:
What about using `typeof<'a>` as the key instead of `typeof<'a>.FullName`?

Reply:
That was what I first tried, but `Type` doesn't support the `comparison` constraint which is required for `Map` keys.